### DISCUSSION TITLE
Currency_Conv merged to development branch

### DISCUSSION THREAD

**Discussion by thawkes70** [thawkes70/Tafe22S219#8] [General]
Hi Chitra,

For your info I have created the Currency_Conv feature and I have committed it back to the Currency_Conv feature branch.  

Cheers

Tony

**Comment by thawkes70** [comment]
Hi Chitra,

FYI I have now merged the Currency_Conv feature with the development branch.

Cheers

Tony

**Reply by Chitusnair** [reply]
Hi Tony,

Since you have merged your feature, when I checkout the development branch, should I be seeing your changes? I dont see it now.

Chitra.

**Reply by thawkes70** [reply]
![image](https://user-images.githubusercontent.com/111416606/188303635-03ddc898-f125-4867-876c-2318afa77f26.png)
